plugins/jreleaser-gradle-plugin/src/main/groovy/org/jreleaser/gradle/plugin/internal/dsl/AbstractGitService.groovy

```diff
@@ -56,6 +56,7 @@ abstract class AbstractGitService implements GitService {
     final Property<String> username
     final Property<String> token
     final Property<String> tagName
+    final Property<String> previousTagName
```

Comment:
Please add a `Property<String> getPreviousTagName()` to the `GitService.groovy` DSL interface.